File: PRAK304-2510817110006-Muhammad Fadhil Lesmana.py
try:
    while True:
        n = int(input())
        if n < 0 or n > 99:
            print("Anda Menginput Melebihi Limit Bilangan")
        elif n == 0:
            print("Nol")
        elif n < 10:
            print("Satuan")
        elif n < 20:
            print("Belasan")
        else:
            print("Puluhan")
except EOFError:
    pass

File: PRAK304-2510817110006-Muhammad Fadhil Lesmana.c
#include <stdio.h>

int main() {
    int n;
    // Membaca input sampai EOF
    while (scanf("%d", &n) != EOF) {
        if (n < 0) {
            printf("Anda Menginput Melebihi Limit Bilangan\n");
        } else if (n == 0) {
            printf("Nol\n");
        } else if (n > 99) {
            printf("Anda Menginput Melebihi Limit Bilangan\n");
        } else if (n < 10) {
            printf("Satuan\n");
        } else if (n < 20) {
            printf("Belasan\n");
        } else if (n < 100) {
            printf("Puluhan\n");
        }
    }
    return 0;
}
